buildViaTravis.sh

```diff
@@ -6,15 +6,15 @@ if [ "$TRAVIS_PULL_REQUEST" != "false" ] || [ "$GRADLE_PUBLISH" == "false" ]; th
   ./gradlew build
 elif [ "$TRAVIS_PULL_REQUEST" == "false" ] && [ "$TRAVIS_TAG" == "" ]; then
   echo -e 'Build Branch with Snapshot => Branch ['$TRAVIS_BRANCH']'
-  ./gradlew -Prelease.travisci=true -PbintrayUser="${bintrayUser}" -PbintrayKey="${bintrayKey}" -PsonatypeUsername="${sonatypeUsername}" -PsonatypePassword="${sonatypePassword}" -Prelease.scope=patch build snapshot
+  ./gradlew -Prelease.travisci=true -PnetflixOss.username=$NETFLIX_OSS_REPO_USERNAME -PnetflixOss.password=$NETFLIX_OSS_REPO_PASSWORD  -Psonatype.signingPassword=$NETFLIX_OSS_SIGNING_PASSWORD -Prelease.scope=patch build snapshot
```

Comment:
These args need to be wrapped in quotes.

Reply:
not necessarily, take https://github.com/Netflix/netflixoss-sample-project-travisci/blob/main/gradle/buildViaTravis.sh#L21 as example

Will add it for consistency with previous versions

Reply:
Thanks.   Ideally these scripts would pass shellcheck, but it there are many gaps in our consistency.  As long as the problem doesn't get worse, I'm okay.